fastlane/Fastfile

```diff
@@ -63,7 +63,8 @@ platform :android do
 
     upload_to_play_store(
       apk: apkPath,
-      track: 'Staging',
+      track: 'production',
```

Comment:
Is the case important here? Just saw it was capitalized before.

Reply:
`production` should be right according to the [official docs](https://docs.fastlane.tools/actions/upload_to_play_store/)

![Screenshot 2024-05-21 at 09 57 30](https://github.com/duckduckgo/Android/assets/1336281/caa40019-59f1-417c-b7d4-7664849a4cd5)